examples/amp-byside-placeholder.amp.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html ⚡>
+<head>
+  <meta charset="utf-8">
+  <title>BySide Placeholder example</title>
+  <link rel="canonical" href="amps.html">
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <style amp-custom>
+    amp-byside-placeholder {
+      color: red;
+    }
+  </style>
+  <script async custom-element="amp-byside-placeholder" src="https://cdn.ampproject.org/v0/amp-byside-placeholder-0.1.js"></script>
+  <script async src="https://cdn.ampproject.org/v0.js"></script>
+</head>
+<body>
+	<h2>Fixed height placeholder content</h2>
+	<amp-byside-placeholder
+		data-webcare-id="D6604AE5D0"
+		data-lang="pt"
+		data-label="amp-simple"
+		layout="fixed-height"
+		height="400"
+	></amp-byside-placeholder>
+
+	<h2>Auto resizable placeholder content</h2>
+	<amp-byside-placeholder
+		data-webcare-id="D6604AE5D0"
+		data-channel=""
+		data-lang="pt"
+		data-fid=""
+		data-label="amp-number"
+		layout="fixed"
+		width="120"
+		height="40"
+		resizable
```

Comment:
Regarding the optional `resizable` attribute: a few points:

1- AMP UX does not support vertically scrollable sub-sections of the page which means the `iframe` must set scrolling to no. With that in mind, I can't see why anyone may not want the resizability feature since otherwise their content will get cut off. It may make sense to always resize without exposing an option.
2- AMP UX prevents content shifting. This means resize requests may not be honoured if resizing causes content shift (for instance if element is in the viewport). Therefore I recommend looking at how `amp-playbuzz` handles providing an "overflow" button which shows up if resize request is denied.